### ISSUE TITLE
BigQuery connector/SP returns "Error: list index out of range" on zero results

### ISSUE THREAD

**Issue by erikspears** [sidprobstein/swirl-search#96]
To see this:

- Setup the new BigQuery SearchProvider (and funding example data)
- Do a search that should **_not_** return results from that data set
- Swirl returns this message to the UI:  `Error: list index out of range`
- And this to the celery-worker.log:
```
[2023-01-24 18:55:02,089: INFO/ForkPoolWorker-10] federate[c5709800-a7c9-4d0b-967a-2c091083bcd8]: BigQuery_8_4: construct_query()
[2023-01-24 18:55:02,090: INFO/ForkPoolWorker-10] federate[c5709800-a7c9-4d0b-967a-2c091083bcd8]: BigQuery_8_4: validate_query()
[2023-01-24 18:55:02,090: INFO/ForkPoolWorker-10] federate[c5709800-a7c9-4d0b-967a-2c091083bcd8]: BigQuery_8_4: execute_search()
[2023-01-24 18:55:04,009: ERROR/ForkPoolWorker-10] BigQuery_8_4: list index out of range
[2023-01-24 18:55:04,010: INFO/ForkPoolWorker-10] BigQuery_8_4: save_results()
[2023-01-24 18:55:04,010: INFO/ForkPoolWorker-10] BigQuery_8_4: Result.create()
```

**Comment by sidprobstein** [comment]
This will be fixed in 1.9

**Comment by sidprobstein** [comment]
Confirmed fix. The issue is that BigQuery returns 1 row when you issue a count query. Fixed now in 1.9 branch.

**Comment by erikspears** [comment]
Validated the fix in a local install of the official 1.9 release, looks great!  Thanks for the quick fix @sidprobstein !
Closing this one out.
